### ISSUE TITLE
Server must synchronize files in default/ with clients

### ISSUE THREAD

**Issue by afwbkbc** [freeorion/freeorion#1401]
Once we encountered a game-breaking bug. I was playing as Gysache and was colonizing planets. They showed +1 or +3 but after I settled these values turned into -1 or -3 and my population died out. After that they started showing +1 and +3 again.
The reason behind it was difference in gysache script between server and my pc, so game thought it's ok to colonize these planets but on server they were not ok, so they died out.
Solution: upon starting game, server must either do some checksum check on default/* files or send all these files to clients (and clients can store it in some per-game or per-server or per-version location).
Bottom line is that these files MUST be the same between server and all clients, otherwise it may result in weird bugs and broken games.
